notebooks/book1/02/anscombes_quartet.ipynb

```diff
@@ -10,38 +10,159 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        from probml_utils import latexify, savefig, is_latexify_enabled
</pre><p>Fix the error raised by this line.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/pyprobml/pull/772/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Done